docs-ref-mapping/reference-unified.yml

```diff
@@ -323,6 +323,16 @@
       href: ~/docs-ref-services/core-http-netty-readme.md
       children:
       - com.azure.core.http.netty*
+    - name: Apache Avro Serializer
+      uid: azure.java.sdk.landingpage.services.sdk.core.ApacheAvroSerialzier
+      href: ~/docs-ref-services/core-serializer-avro-apache-readme.md
+      children:
+      - com.azure.core.serializer.avro.apache
+    - name: Tracing
```

Comment:
This should go under `core` imo. It is for the other 3 languages.

Reply:
Tracing is under core. It is parallel to serializer. They all belongs to core if you scroll up a bit.